server/routes/reports.js

```diff
@@ -0,0 +1,20 @@
+const router = require('express').Router();
+const Rents = require('../models/Rent');
+
+router.get('/report', async (_, res) => {
+  try {
+    res.json(await Rents.find()
```

Comment:
This looks like a getAll that's already implemented

Reply:
In this case i need the find instead, because the get all would filter the one's with status: false, and since it's a report i need every rent. The status:false in rent is to make it disappear from the selection.

Reply:
makes sense